app/processing/get-completed-payment-requests-filter.js

```diff
@@ -13,7 +13,6 @@ const getCompletedPaymentRequestsFilter = (paymentRequest) => {
       return {
         schemeId: paymentRequest.schemeId,
         frn: paymentRequest.frn,
-        marketingYear: paymentRequest.marketingYear,
```

Comment:
Why was this removed here?

Reply:
The reason for the ticket is that a CS payment request can have multiple marketing years in it's invoice lines.  One of them will be shown in the payment request level property sure, but if we restrict the filter to only those with the same year, we may risk missing a previous payment request because the year on the payment request happened to be different.

Fortunately, the way CS works, we can use the contract number and FRN to still get all previous payment requests.

Reply:
Ah okay, good :)